projects/pastanaga/src/lib/dropdown/dropdown.component.ts

```diff
@@ -14,7 +14,7 @@ import { PopupService } from '../popup/popup.service';
 @Component({
     selector: 'pa-dropdown',
     templateUrl: './dropdown.component.html',
-    styleUrls: ['../popup/_popup.scss', './_dropdowns.scss'],
+    styleUrls: ['./_dropdowns.scss'],
```

Comment:
_dropdowns.scss is imported in several components, it would make sense to import it in styles/common-reset so it is globally available in Pastanaga style, no?

Reply:
probably yes, I'll have a look